lib/JSON2CSVBase.js

```diff
@@ -181,7 +181,7 @@ class JSON2CSVBase {
     //a backslash, and it's not at the end of the stringifiedValue.
     stringifiedValue = stringifiedValue
       .replace(/^"(.*)"$/, this.opts.quote + '$1' + this.opts.quote)
-      .replace(/(\\")(?=.)/g, this.opts.doubleQuote)
+      .replace(/(\\")(?!$)/g, this.opts.doubleQuote)
```

Comment:
Assuming your tests pass, this fix is good with me. Although I thought `$`  would match both end of line and end of string. Or does it only match end of string?

Reply:
Looks like `$` only matches line break if the multiline flag (`m`) is true (which it's not). So this looks good to me.